src/main.rs

```diff
@@ -146,6 +146,14 @@ fn main() {
 								.entry(MenuItem::new("Menu item with something on the\tright"))
 							// menus are currently commented out in the floem codebase
 						})
+						.on_resize(move |r| {
+							window_size_config.general.write().window_settings.window_size =
+								(r.x1, r.y1);
+						})
+						.on_event(EventListener::WindowClosed, move |_| {
```

Comment:
This doesn't capture cmd+q :( 
It works for alt+f4 though. Need to look into it more.

Reply:
Implemented a debounce primitive. Closing this, it will now save 100ms after the last value sent.